client/src/Landing.css

```diff
@@ -7,11 +7,11 @@
     background-attachment: fixed;
     background-position: center; 
     z-index: 9999;
-    background-image: url("./assets/img/Austin-Greenbelt.jpg");
+    background-image: url("./assets/img/Test.jpg");
```

Comment:
Reason why this background file is called 'test'?

Reply:
was just testing out new photos and never changed it haha